manage.py

```diff
@@ -1041,6 +1043,11 @@ def _add_terraform_subparser(subparsers):
         help=ARGPARSE_SUPPRESS,
         nargs='+')
 
+    tf_parser.add_argument(
+        '--cluster',
+        help=ARGPARSE_SUPPRESS,
+        nargs='+')
```

Comment:
Recommendation: use the `action=UniqueSetAction` keyword arg to ensure this is always a set of unique clusters with no duplicates. I know you cast this to a `set` down the line (when you do `clusters = set(options.cluster or config.clusters())`), but using this action will avoid requiring that the caller knows this could contain dupes. Setting the `default=set()` is also a good idea.